charts/rstudio-connect/NEWS.md

```diff
@@ -1,3 +1,7 @@
+# 0.2.35
+
+- Add the ability to set annotations to the PVC.
```

Comment:
Here and elsewhere: avoid using acronyms like "PVC" in user-facing docs. We should use some variant of "Persistent Volume Claim" instead.

Reply:
```suggestion
- Add the ability to set annotations to the Persistent Volume Claim.
```